styles/_main.scss

```diff
@@ -0,0 +1,74 @@
+.outer-container {
+  margin: 3rem auto;
+}
+
+.row {
+  width: $max-content-width;
+
+  margin: auto;
+}
+
+.content-photo {
+  display: flex;
+  flex-direction: column;
+
+  padding-bottom: $gutter;
+
+  &__container {
+    display: flex;
+    flex-direction: row;
+
+    &__image {
+      display: flex;
+      flex-grow: 2;
```

Comment:
Don't rely on `flex-grow` to divy up `widths`. `flex-grow` only dictates how remaining whitespace is alotted AFTER initial rendering (how do we divide the tickets we have left)